doc/source/whatsnew/v1.0.0.rst

```diff
@@ -210,6 +210,7 @@ Groupby/resample/rolling
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
 -
+- Bug in :meth:`Resampler.size()` and `Resampler.count()` returning wrong `dtype` when used with an empty series or dataframe (:issue:`28427`)
```

Comment:
I don't think you need parens in the :meth:`...`. Not sure if having them cause issues.

dtype doesn't need to be in backticks (it's not a parameter)

Reply:
Removed the parens, removed backyticks around `dtype` and added `:meth:` in front of `Resampler.count`.